grails-app/views/order/shipOrder.gsp

```diff
@@ -141,7 +162,7 @@
 													   name="shipOrderItems[${status}].quantityToShip"
 													   min="${shipOrderItem.quantityMinimum}"
 													   max="${shipOrderItem?.quantityMaximum}"
-													   value=""/>
+													   value="${shipOrderItem?.quantityToShip ?: ''}"/>
```

Comment:
It should be "value=""/>" so it defaults to blank.

Reply:
I shoud default to blank, but only on first enter, after the import that it should take the imported value. I see that this always defaults to shipOrderItem?.quantityToShip (even without any data). Ill fix that.